sed/config/flash_example_config.yaml

```diff
@@ -19,12 +19,23 @@ core:
   # year: 20xx
 
 dataframe:
+  # The name of the DAQ system to use. Necessary to resolve the filenames/paths.
+  daq: fl1user3
   # The offset correction to the pulseId
   ubid_offset: 5
   # the number of iterations to fill the pulseId forward. 
   forward_fill_iterations: 2 
-  # The name of the DAQ system to use. Necessary to resolve the filenames/paths.
-  daq: fl1user3
+  # if true, removes the 3 bits reserved for dldSectorID from the dldTimeandSector column
+  unravel_8s_detector_time_channel: True
+  time_step_size: 0.16460905596613884 # 0.020576131995767355
```

Comment:
This already exists, and is called tof_binwidth
Also, you should not change it, but rather the parameter tof_binning. Have a look at the code that does the tof->ns conversion in the energy calibration, 
https://github.com/OpenCOMPES/sed/blob/e3af1451631bc0aeac68c2833e50841de71c6091/sed/calibrator/energy.py#L2056

Reply:
I'll fix the name duplicates to follow MPES convention, but the tof_binning brings up a question: what about jittering? is this done before and so ignores the tof_binning?

Reply:
also, why is binning used in the power of 2?
This means in our case we would need binning of 3, not 8. correct? 
If so, I find this very confusing.

Reply:
but why limiting binning to powers of 2?

Reply:
even more confusing, why binning-1?
I just don't understand the logic of this.
would it not be simpler to have tof_binning to be the value you multiply to get the right units? `ns = step * tof_binwidth * tof_binning`

Reply:
This is how binning is defined in the tdc config file

Reply:
> but why limiting binning to powers of 2?

Because that's what digital binning in digital detectors does, sequentially aggregating pixels, or shifting bits

Reply:
I agree this is all somewhat inconsistent, and apparently also only specific to our instrument. This is what is written in our TDC config:
```
[EVENTFORMAT]
; THIS section is read only by the sctdcDeviceWrapper library
; it has no effect with the pure scTDC library
; binnings are specified in powers of 2, i.e. bin=0 means no binning
```
I.e. these settings there are pure powers of 2, without the -1. My initial idea was to make the code compatible, so that you could just copy these values into the sed config.
Then I think I mixed up the definition of binning in EPICS Areadetector, which uses the actual binning values, and thought binning=1 should not do any binning, that's how the -1 sneaked in.

If you look at the documentation of the TDC interface library, it also allows binning, and these parameters are given as the actual binning values, i.e. binning=1 does not binning, but it states that only powers of 2 are allowed.

So, I see two ways forward. 
1. We remove all the -1s, and have tof_binning be powers of 2, like in our tdc ini file
2. We define binning as the actual binning values, and add a note that only powers of 2 are allowed (?)

Reply:
Seems to be even worse. The additional TDC config parameter HardwareBinningT seems to be given as absolute binning number, as far as I understand (no documentation available)... So, option 2 seems to be the most consistent I would say.

Reply:
Option 2 would be the most logical, in my opinion.

Reply:
I implemented this with the power version for now, and would leave it like this. We should do an other PR to refactor everything to option 2. What do you think?

Reply:
fine with me, open an issue then.

Reply:
FYI, my current interpretation has no -1 (as in the energy calibration function from mpes, if I read correctly.)

Reply:
You are right, this is inconsistent at so many places. I can look into this and fix this soon.